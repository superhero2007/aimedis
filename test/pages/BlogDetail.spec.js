import React from 'react';
import { shallow } from 'enzyme';

import BlogDetail from 'pages/BlogDetail';

function setup() {
  return shallow(<BlogDetail />);
}

describe('BlogDetail', () => {
  const wrapper = setup();

  it('should be a StatelessComponent', () => {
    expect(wrapper.instance()).toBeNull();
  });

  it('should render properly', () => {
    expect(wrapper.html()).toMatchSnapshot();
  });
});
