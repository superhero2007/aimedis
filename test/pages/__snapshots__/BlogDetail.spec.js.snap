// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BlogDetail should render properly 1`] = `"<div class=\\"app__blog__detail app__route\\"><p class=\\"d-none\\">1</p><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__blog__detail__image\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></div><div class=\\"container\\"><section class=\\"app__blog__detail__section1\\"><div class=\\"row\\"><div class=\\"col-12\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><p class=\\"app__blog__detail__section1__title\\">AR and VR are the new tools in your doctor’s bag</p></div></div><div class=\\"col-12 col-lg-8 offset-lg-2 app__blog__detail__section1__content\\"><div class=\\"app__blog__detail__section1__content__info\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__blog__detail__section1__content__info__share\\"><div class=\\"app__blog__detail__section1__content__info__share__element\\"><div class=\\"app__blog__detail__section1__content__info__share__element__icon\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><div class=\\"app__blog__detail__section1__content__info__share__element__text\\">Chris Welch</div></div><div class=\\"app__blog__detail__section1__content__info__share__element\\"><div class=\\"app__blog__detail__section1__content__info__share__element__icon\\"><img src=\\"\\" alt=\\"No Asset\\"/></div><div class=\\"app__blog__detail__section1__content__info__share__element__text\\">2 WEEKS AGO</div></div></div></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__blog__detail__section1__content__info__link d-none d-lg-flex\\"><a href=\\"#\\" class=\\"app__blog__detail__section1__content__info__link__element\\"><img src=\\"\\" alt=\\"No Asset\\"/></a><a href=\\"#\\" class=\\"app__blog__detail__section1__content__info__link__element\\"><img src=\\"\\" alt=\\"No Asset\\"/></a><a href=\\"#\\" class=\\"app__blog__detail__section1__content__info__link__element\\"><img src=\\"\\" alt=\\"No Asset\\"/></a></div></div></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><hr/></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><p class=\\"app__blog__detail__section1__content__description first__letter\\">Medicine has always been an obscure discipline to practise doctor has been looking at with awe and respect, and until quite recently, was often an unapproachable person. However, this is changing rapidly. He is a service provider and you, the patient, are a consumer, and thus protected by the prevailing consumer laws.</p></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><p class=\\"app__blog__detail__section1__content__subtitle\\">Being good at medicine is not good enough</p></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><p class=\\"app__blog__detail__section1__content__description1\\">160 years ago, in 1858, Henry Gray published Gray’s Anatomy – the bible for every doctor. Back then, body snatchers were a prime source of obtaining cadavers for dissections. About 70 years later, the first antibiotic was discovered in the form of penicillin. Fast forward another 40 years, and people were still dying from simple infections. Today, these things would sound so alien! No antibiotics, illegally obtained bodies for dissection, people dying from simple infection rather than grave, life-threatening diseases. Medicine has come a long way since the time of Mr Gray (Surgeons use a Mr instead of Dr in the UK).</p></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><p class=\\"app__blog__detail__section1__content__description1\\">Today, a doctor is not expected to do his job, he has to be in a sensitive manner, warranted by the profession. He needs to have proper bedside manners in most of the medical schools. However, in the age of technology, this is not enough!</p></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__blog__detail__section1__content__reference\\"><p class=\\"app__blog__detail__section1__content__reference__text\\">«Today, medical schools have started recruiting the importance of telemedicine in the healthcare ecosystem.»</p><p class=\\"app__blog__detail__section1__content__reference__speaker\\">Irina Danilenko – doctor</p></div></div></div><div class=\\"col-12 d-none d-lg-block app__blog__detail__section1__content__feature\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><img src=\\"\\" alt=\\"No Asset\\"/></div></div><div class=\\"col-12 col-lg-6 offset-lg-3 d-none d-lg-block\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><p class=\\"app__blog__detail__section1__content__description1\\">Medical advances on the fronts of new drug discoveries have slowed down. Instead, pharmaceuticals are being used to improve their chances. The next big thing is biologicals and personalized medicine.</p></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><p class=\\"app__blog__detail__section1__content__description1\\">However, when it comes to the doctor’s clinic, a lot of things are changing as well. He has the tools to capture the patient’s data without having to spend on his computer. With IoT devices and integrated software, things are becoming easier for doctors. That said, there is still a huge scope for technology in his clinic.</p></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__blog__detail__section1__content__list\\"><ul><li>Fashion</li><li>Design</li><li>Art</li><li>Marketing</li><li>Management</li><li>Communications</li></ul></div></div></div><div class=\\"col-12\\"><div class=\\"app__blog__detail__section1__content__video\\"><div class=\\"app__video\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"video-react-controls-enabled video-react-paused video-react-fluid video-react-user-active video-react-workinghover video-react\\" style=\\"padding-top:56.25%\\" role=\\"region\\" tabindex=\\"-1\\"><video class=\\"video-react-video\\" preload=\\"auto\\" poster=\\"\\" src=\\"http://media.w3.org/2010/05/bunny/movie.mp4\\" tabindex=\\"-1\\"></video><div class=\\"video-react-loading-spinner\\"></div><button class=\\"video-react-big-play-button video-react-big-play-button-center big-play-button-hide\\" type=\\"button\\" aria-live=\\"polite\\" tabindex=\\"0\\"><span class=\\"video-react-control-text\\">Play Video</span></button><div class=\\"video-react-control-bar video-react-control-bar-auto-hide\\"><button class=\\"video-react-play-control video-react-control video-react-button video-react-paused\\" type=\\"button\\" tabindex=\\"0\\"><span class=\\"video-react-control-text\\">Play</span></button><div class=\\"video-react-progress-control video-react-control\\"><div class=\\"video-react-progress-holder video-react-slider-horizontal video-react-slider\\" tabindex=\\"0\\" role=\\"slider\\" aria-label=\\"video progress bar\\" aria-valuenow=\\"NaN\\" aria-valuetext=\\"0:00\\" aria-valuemin=\\"0\\" aria-valuemax=\\"100\\"><div data-current-time=\\"0:00\\" class=\\"video-react-play-progress video-react-slider-bar\\" style=\\"width:NaN%\\"><span class=\\"video-react-control-text\\">Progress: NaN%</span></div></div></div><div class=\\"video-react-current-time video-react-time-control video-react-control\\"><div class=\\"video-react-current-time-display\\" aria-live=\\"off\\"><span class=\\"video-react-control-text\\">Current Time </span>0:00</div></div><div class=\\"video-react-time-control video-react-time-divider\\" dir=\\"ltr\\"><div><span>/</span></div></div><div class=\\"video-react-duration video-react-time-control video-react-control\\"><div class=\\"video-react-duration-display\\" aria-live=\\"off\\"><span class=\\"video-react-control-text\\">Duration Time </span>0:00</div></div><div class=\\"video-react-volume-menu-button-horizontal video-react-vol-3 video-react-volume-menu-button video-react-menu-button-inline video-react-control video-react-button video-react-menu-button\\" role=\\"button\\" tabindex=\\"0\\"><div class=\\"video-react-menu\\"><div class=\\"video-react-menu-content\\"><div class=\\"video-react-volume-bar video-react-slider-bar video-react-slider-horizontal video-react-slider\\" tabindex=\\"0\\" role=\\"slider\\" aria-label=\\"volume level\\" aria-valuenow=\\"100.00\\" aria-valuetext=\\"100.00%\\" aria-valuemin=\\"0\\" aria-valuemax=\\"100\\"><div class=\\"video-react-volume-level\\" style=\\"width:100.00%\\"><span class=\\"video-react-control-text\\"></span></div></div></div></div></div><button class=\\"video-react-icon-fullscreen video-react-fullscreen-control video-react-control video-react-button video-react-icon\\" type=\\"button\\" tabindex=\\"0\\"><span class=\\"video-react-control-text\\">Non-Fullscreen</span></button></div></div></div></div></div><div class=\\"app__blog__detail__section1__content__share\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__blog__detail__section1__content__share__text\\">Like it? Share it!</div></div><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__blog__detail__section1__content__share__link\\"><a href=\\"#\\" class=\\"app__blog__detail__section1__content__share__link__element\\"><img src=\\"\\" alt=\\"No Asset\\"/></a><a href=\\"#\\" class=\\"app__blog__detail__section1__content__share__link__element\\"><img src=\\"\\" alt=\\"No Asset\\"/></a><a href=\\"#\\" class=\\"app__blog__detail__section1__content__share__link__element\\"><img src=\\"\\" alt=\\"No Asset\\"/></a></div></div></div></div></div></section><section class=\\"app__blog__detail__section2\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__blog__detail__section2__title\\">You might like these articles too</div></div><div class=\\"app__blog__detail__section2__content\\"><div class=\\"row\\"><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/1\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"/></div><p class=\\"app__blog__item__content__title\\">Why Japan, the Tech giant, still needs to improve Health Tech</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/2\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"/></div><p class=\\"app__blog__item__content__title\\">Why diagnosing leprosy in the Philippines is not a big problem any more: Learns</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div><div class=\\"col-12 col-lg-4\\"><a href=\\"/blog/3\\"><div class=\\"animated\\" style=\\"animation-duration:1s;opacity:0\\"><div class=\\"app__blog__item\\"><div class=\\"app__blog__item__content\\"><div class=\\"app__blog__item__content__img\\"><img src=\\"\\" alt=\\"No Item Asset\\"/></div><p class=\\"app__blog__item__content__title\\">Why diagnosing leprosy in the Philippines is not a big problem any more: Learns</p><p class=\\"app__blog__item__content__description\\">Aimedis supports patients and doctors by making data available quickly and reliable which results in a quick supports patients and doctors by</p></div></div></div></a></div></div></div></section></div></div>"`;
